swayfmt/tests/mod.rs

```diff
@@ -3129,3 +3130,29 @@ fn impl_func_where() {
         "#},
     );
 }
```

Comment:
Maybe it's just my paranoia with multiple changes in one go, but it may be worth to duplicate the following test but with different spacing - i.e no spacing between the hex value and `ContractId`. All good if not otherwise.

Reply:
Great idea, thanks. I've updated the test.